docs/how-to-translate-files.md

```diff
@@ -21,6 +21,9 @@ Each file and folder will show a progress bar. The **blue** portion of the progr
 
 Select a file to work on and Crowdin will open the editor view.
 
+> [!NOTE]
+> When the editor view opens, you will need to click the settings icon (shown as a gear) and switch the 'HTML Tags Displaying' setting to 'Show'. This will ensure you can see tags such as `<code></code>` instead of `<0></0>`.
```

Comment:
```suggestion
> When the editor view opens, you will need to click the settings icon (shown as a gear) and switch the 'HTML tags displaying' setting to 'SHOW'. This will ensure you can see tags such as `<code></code>` instead of `<0></0>`.
```

Reply:
I could have *sworn* it was title cased in the settings menu when I looked an hour ago.

Spoiler: It's not. 🤦🏻